javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentTracerProviderConfigurer.java

```diff
@@ -20,6 +20,7 @@
 @AutoService(AutoConfigurationCustomizerProvider.class)
 public class AgentTracerProviderConfigurer implements AutoConfigurationCustomizerProvider {
   private static final String ADD_THREAD_DETAILS = "otel.javaagent.add-thread-details";
+  private static final String ADD_BAGGAGE = "otel.javaagent.span.add-baggage";
```

Comment:
this would correspond with, e.g. `otel.traces.exporter`, `otel.metrics.exporter`, `otel.logs.exporter`

```suggestion
  private static final String ADD_BAGGAGE = "otel.javaagent.traces.add-baggage";
```